### PR TITLE
nov dev update

### PR DESCRIPTION
**Pull request by korydraughn**

### CODE REVIEW

File: content/posts/irods-development-update-november-2023.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+We're finally back from Supercomputing and it went very well. For those who attended, we hope you had as good a time as us. For those who did not attend, maybe we'll see you next year.
+
+Now for this month's update.
+
+With [iRODS 4.3.1 finally released](https://irods.org/2023/10/irods-4-3-1-is-released/), we're going to focus the beam on projects surrounding the server. That means addressing long standing issues in plugins and clients.
+
+First up is the iRODS S3 API. We've improved support for the MinIO client, added a configuration option which allows administrators to define the region, and added support for the `GetObjectLockConfiguration` API operation. We've also containerized the tests around the project. Now anyone can run the tests and build confidence using only `docker compose`.
+
+For the iRODS HTTP API, we're investigating alternative designs for OpenID Connect. Specifically, we're looking at making the HTTP API act as a resource server instead of an OAuth client. For more information, see issue [#155](https://github.com/irods/irods_client_http_api/issues/155).
```
[the remark this answers is not in the document]
Comment (korydraughn) [reply content/posts/irods-development-update-november-2023.md]: I'm not fond of how I linked the release articles.

Let me see if I can come up with something better.

[the remark this answers is not in the document]
Comment (korydraughn) [reply content/posts/irods-development-update-november-2023.md]: Updated.

File: content/posts/irods-development-update-november-2023.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+Hello Everyone,
+
+We're finally back from Supercomputing and it went very well. For those who attended, we hope you had as good a time as us. For those who did not attend, maybe we'll see you next year.
+
+Now for this month's update.
+
+With [iRODS 4.3.1 finally released](https://irods.org/2023/10/irods-4-3-1-is-released/), we're going to focus the beam on projects surrounding the server. That means addressing long standing issues in plugins and clients.
+
+First up is the iRODS S3 API. We've improved support for the MinIO client, added a configuration option which allows administrators to define the region, and added support for the `GetObjectLockConfiguration` API operation. We've also containerized the tests around the project. Now anyone can run the tests and build confidence using only `docker compose`. v0.1.0 was released earlier this month. Click [here](https://irods.org/2023/11/initial-release-of-the-irods-s3-api/) to read all about it.
```
Comment (trel) [inline content/posts/irods-development-update-november-2023.md:16]: please just make 'v0.1.0 was released earlier this month' the link to the news, no need for the last sentence.

File: content/posts/irods-development-update-november-2023.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+For the iRODS HTTP API, we're investigating alternative designs for OpenID Connect. Specifically, we're looking at making the HTTP API act as a resource server instead of an OAuth client. For more information, see issue [#155](https://github.com/irods/irods_client_http_api/issues/155). v0.1.0 was released earlier this month. Click [here](https://irods.org/2023/11/initial-release-of-the-irods-http-api/) to read all about it.
```
Comment (trel) [inline content/posts/irods-development-update-november-2023.md:18]: same here.

Comment (korydraughn) [reply content/posts/irods-development-update-november-2023.md]: done

**Comment by trel** [comment]
squash em

**Comment by korydraughn** [comment]
Squashed.

**Pull request merged**